### DISCUSSION TITLE
Search not working with for translated products

### DISCUSSION THREAD

**Discussion by thomasbrelet** [Shopify/storefront-api-feedback#134] [Help]
Hello, 

I use the `@inContext(language)` feature on storefront API to get translated content and it's working great. But if I try to search products using the "query" parameter it's not working in the specified language. The query parameter only works in the original language. 

The results are on the context language but I have to query in the origin language.

Examples:

```
@inContext(language=EN)
Storefront.client.product.fetchQuery({first:10,query:"title:Dress"}).then(product=>(product))
=> Returns results

@inContext(language=FR)
Storefront.client.product.fetchQuery({first:10,query:"title:Robe"}).then(product=>(product))
=> NO RESULT

@inContext(language=FR)
Storefront.client.product.fetchQuery({first:10,query:"title:Dress"}).then(product=>(product))
=> Returns results translated in French (Product with title "Robe" which is "Dress" in French)
```

Several people have asked the question on the Shopify forum but I haven't seen any answers

**Comment by fahjan** [comment]
any updates on this case?
@thomasbrelet

**Reply by thomasbrelet** [reply]
No, we are still looking for a solution...
